spec/dry_run_spec.rb

```diff
@@ -1,10 +1,6 @@
 require 'spec_helper'
 
 describe Slackistrano do
-  before(:each) do
-    Rake::Task['load:defaults'].execute
```

Comment:
How are you running the specs? I don't get any errors when running `rspec spec/dry_run_spec.rb`.